src/components/groups/group.middlewares.ts

```diff
@@ -7,7 +7,7 @@ export const joinGroup = async (req: Request, res: Response, next: NextFunction)
   const user = req.user as User
   const group = req.group
 
-  if (!group.doNotDisturb && !group.users?.includes(user)) {
+  if (!group.users?.includes(user)) {
```

Comment:
Don't remove this check, since we don't want to allow users to join closed groups.

Reply:
Misunderstood the context, gonna mend it.